### PR TITLE
Fix viewcell margins on listview

### LINKED ISSUE #4719: ListView Margins inside ViewCells on P13 are broken

### PR DESCRIPTION
**Pull request by PureWeen**
### Description of Change

Use `IView.Measure/Arrange` instead of calling it directly on the platform.
If this gets the @hartez  stamp of approval then I'll apply the same logic to these two

https://github.com/dotnet/maui/issues/4762
https://github.com/dotnet/maui/issues/5621

### Issues Fixed

Fixes #4719

### CODE REVIEW

File: src/Core/src/Handlers/ViewHandlerExtensions.Android.cs [hunk 1]
```diff
@@ -1,11 +1,68 @@
...
+using Microsoft.Maui.Platform;
+using System;
+using static Android.Views.View;
 
 namespace Microsoft.Maui
 {
-	public static partial class ViewHandlerExtensions
+	internal static partial class ViewHandlerExtensions
 	{
+		internal static void LayoutVirtualView(
```
Comment (PureWeen) [inline src/Core/src/Handlers/ViewHandlerExtensions.Android.cs:12]: Copied this code out of each platforms `LayoutView`. I put these into extension methods so I can reuse them everywhere where I'm hosting an IView inside a platform control

**Review by hartez — APPROVED** [review]

File: src/Core/src/Handlers/ViewHandlerExtensions.Android.cs (hunk 1 above)
Comment (hartez) [reply src/Core/src/Handlers/ViewHandlerExtensions.Android.cs:12]: Probably worth creating a reminder to review these and the existing LayoutView/LayoutViewGroup/LayoutPanel stuff to see if we can consolidate it all into one spot.

**Pull request merged**